perm.go

```diff
@@ -0,0 +1,10 @@
+package main
+
+import "github.com/google/uuid"
+
+func hasReadPerm(username string, uuid uuid.UUID) bool {
```

Comment:
why not use the original permission checking function?

Reply:
This is just a placeholder. I want to use the original permission checking function.